CHANGELOG.md

```diff
@@ -126,11 +96,64 @@
 
 * Fixes stale hostnames bug in prepared statements after `master_update_node`
 
+* Fixes the relation size bug during rebalancing
+
+* Fixes two race conditions in the get_rebalance_progress
+
 * Fixes using 2PC when it might be necessary
 
 * Preserves access method of materialized views when undistributing
```

Comment:
maybe move `reserves access method of materialized views when undistributin` on top of `Fixes..` block?

Reply:
I intentionally did not reorder the elements that were introduced in #5043 but this makes sense to me, so I'll do it.